flashinfer/__main__.py

```
@@ -216,6 +258,152 @@ def clear_cubin_cmd():
+    click.secho("Wheel index:", fg="magenta", nl=False)
+    click.secho(f" {resolved_index_url}", fg="cyan")
+    click.secho("Requirement:", fg="magenta", nl=False)
+    click.secho(f" {requirement}", fg="cyan")
+    click.secho("Command:", fg="magenta", nl=False)
+    click.secho(f" {' '.join(cmd)}", fg="cyan")
+
+    if dry_run:
+        click.secho("Dry run requested; pip install was not executed.", fg="yellow")
+        return
+
+    result = subprocess.run(cmd, check=False)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)